### ISSUE TITLE
Any chance that we can set typography field's kirki-host-font-locally to enabled by default?

### ISSUE THREAD

**Issue by davidwerks** [themeum/kirki#1911]
### Issue description:

Any chance that we can set typography field's **kirki-host-font-locally** checkbox to enabled by default?

### Version used:
Version 3.0.33

![screen shot 2018-05-30 at 14 57 38](https://user-images.githubusercontent.com/18698470/40726593-efc506b0-641d-11e8-9726-49a1835f9b3d.png)

**Comment by aristath** [comment]
I actually tried that in a previous version... The result was that I had to release 6 updates in 2 days in order to resolve issues with this forced method.

Not all hosts are properly configured. A wrong folder permission at the wrong place and fonts don't load at all, or worse it all comes down crashing.
I solved the permissions issue etc, all the technical stuff has been resolved. 
But that still leaves another issue - and this one is a moral issue and not a technical one. Which makes this worse.

When this option is enabled, it downloads fonts and saves them to the server. No plugin or theme should EVER write to the filesystem without the user's knowledge. Users should know what happens. 
So instead of having this option on by default it is off, and users who want to do that can simply enable the option.

It was added because in the google-fonts repository on github there was a [big discussion about GDPR ](https://github.com/google/fonts/issues/1495) and there was no solution in the horizon.
If the option is on by default and the user's server is misconfigured or if the apache www-data user simply doesn't have permissions to write to the wp-content/uploads folder, then the user may have a false sense of security when it comes to GDPR. Having it optional forces users to turn the checkbox on. If it doesn't work, they'll notice it and investigate or just disable it. But if it's on by default, then both theme developers and myself are going to get hammered by support tickets from people saying that their fonts are broken when in reality their webservers are not properly configured.

I hope the above explanation makes sense.
Sure it's possible to force the option on, if you dig around in the code there are plenty of workarounds.
But I'd advise against it. There's a really good reason it's set to off by default.
